service/framework/kvmconfigv2/resources.go

```diff
@@ -0,0 +1,185 @@
+package kvmconfigv2
+
+import (
+	"github.com/cenkalti/backoff"
+	"github.com/giantswarm/certs"
+	"github.com/giantswarm/kvm-operator/service/cloudconfigv2"
+	"github.com/giantswarm/kvm-operator/service/resource/configmapv2"
+	"github.com/giantswarm/kvm-operator/service/resource/deploymentv2"
+	"github.com/giantswarm/kvm-operator/service/resource/ingressv2"
+	"github.com/giantswarm/kvm-operator/service/resource/namespacev2"
+	"github.com/giantswarm/kvm-operator/service/resource/pvcv2"
+	"github.com/giantswarm/kvm-operator/service/resource/servicev2"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+	"github.com/giantswarm/operatorkit/framework/resource/metricsresource"
+	"github.com/giantswarm/operatorkit/framework/resource/retryresource"
+	"github.com/giantswarm/randomkeys"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	ResourceRetries uint64 = 3
+)
```

Comment:
For now I copied this const everywhere where it is needed. I think it would be good to move it to the operatorkit though.